app/dashboard/templates/process_bounty.html

```diff
@@ -57,6 +57,7 @@ <h3>{% trans "Basic Payout" %}</h3>
                 </div>
                 <div class="w-100 my-2">
                   <label for=bountyFulfillment>{% trans "Bounty Submission:" %}</label>
+                  <label id="tooltip">Where is my Eth going? <i class='fa fa-info-circle'></i></label>
```

Comment:
Can you wrap all of the copy throughout this file in `{% trans "copy here" %}`

Reply:
Yeah okay.